tests/qtopengl/CMakeLists.txt

```diff
@@ -44,13 +44,12 @@ create_test_sourcelist(testDriver qtopengltests.cpp ${testSrcs})
 
 add_executable(AvogadroQtOpenGLTests ${testDriver})
 
-qt5_use_modules(AvogadroQtOpenGLTests OpenGL)
-
 target_link_libraries(AvogadroQtOpenGLTests
   AvogadroQtOpenGL
   vtkImagingCore
   vtkIOImage
-  vtkRenderingQt)
+  vtkRenderingQt
+  Qt5::OpenGL)
```

Comment:
Are you sure this is required? I factored out the need for the Qt5 OpenGL module as it is deprecated and moved to the new QOpenGL* classes.

Reply:
I can try without, this is just the result of merging the qt5_use_modules in here.